R/downloadFile.R

```diff
@@ -103,7 +103,8 @@ downloadFileButton <- function(id,
 #' Server-side function for the downloadFileButton.  This is a custom
 #' high-functionality button for file downloads supporting single or multiple
 #' download types.  The server function is used to provide the data for download.
-#' @param id string represents the module id
+#' @param ... free parameters list for shiny to pass session variables based on the module call(session, input, output)
+#'  variables. \emph{Note}: User must send first argument only which is string represents the module id
```

Comment:
Can you update the verbiage as following - The first argument of this function must be the ID of the Module's UI element

Reply:
done